homestar-runtime/tests/cli.rs

```diff
@@ -309,3 +315,257 @@ fn test_daemon_serial() -> Result<()> {
 
     Ok(())
 }
+
+#[test]
+#[serial]
+#[cfg(windows)]
+fn test_server_not_running_serial() -> Result<()> {
```

Comment:
with some overlap on the tests, maybe we wrap some of these into functions and use them across windows/non-windows runs?